sonarqube-companion-rest/src/main/java/pl/consdata/ico/sqcompanion/config/model/GroupDefinition.java

```diff
@@ -21,5 +21,7 @@
     private List<GroupEvent> events;
     @Singular
     private List<WebhookDefinition> webhooks;
+    @Singular
+    private List<String> members;
```

Comment:
dla memberów widzę potrzebę dodatkowych informacji:
- aliasy - alternatywne user id związane z tym samym użytkownikiem (w sonarze mogą występować różne definiujące tą samą osobę)
- daty przynależności do grupy (minimum to początek dołączenia, koniec pewnie nie jest potrzebny bo wystarczy gościa usunąć z grupy)
- pewnie możliwość zdefiniowania imienia i nazwiska żeby nie polegać tylko na id i email

to pewnie kwalifikuje już do stworzenia sekcji users (z definicją aliasów, imienia i nazwiska) i linkowania tutaj jako przynależność (ref + data przyłączenia)

Reply:
Racja.

Reply:
Jeśli chodzi o przynależność do group, to ja bym to zrobił w tej nowej sekcji, tak żeby można było od razu dodać/usunąć do/z paru grup bez potrzeby skakania po drzewku. Zwłaszcza, że jak wspomniałeś user może mieć kilka aliasów.

Reply:
ważne też na pewno są daty przynależności żeby liczyć naruszenia w kontekście pracy w zespole a nie że np. ja przyjdę z bagażem 10k+ 🤣